Gemfile

```diff
@@ -15,6 +15,7 @@ group :test do
   gem "dry-struct", require: false, github: "dry-rb/dry-struct"
   gem "i18n", "1.8.2", require: false
   gem "transproc"
+  gem "json-schema"
```

Comment:
The gem supports v6. The reason I went for it was because I could not find out how to validate the produced schema against the meta schema in JSONSchemer. Do you know if that is possible?

Reply:
Just load metaschema as a json schema and try to validate produced schema? Metaschema is just a json-schema for validating other json-schemas after all. But from a quick glance we should provide metaschema ourselves.

But if `json_schema` actually supports V6, I guess it's fine as is.